MdeModulePkg/Universal/Variable/RuntimeDxe/VariableStandaloneMm.c

```diff
@@ -87,21 +87,3 @@ VariableServiceInitialize (
 {
   return MmVariableServiceInitialize ();
 }
-
-/**
-  Whether the TCG or TCG2 protocols are installed in the UEFI protocol database.
-  This information is used by the MorLock code to infer whether an existing
-  MOR variable is legitimate or not.
-
-  @retval TRUE  Either the TCG or TCG2 protocol is installed in the UEFI
-                protocol database
-  @retval FALSE Neither the TCG nor the TCG2 protocol is installed in the UEFI
-                protocol database
-**/
-BOOLEAN
-VariableHaveTcgProtocols (
-  VOID
-  )
-{
-  return FALSE;

```

Comment:
I have no objections to the PR as-is. However, given that the only purpose of `VariableHaveTcgProtocols()` appears to be this check, and the conditional update of the variable, we could also just return TRUE here, no? We might want to rename this function to make it more obvious what the narrow use-case is for which it exists.

Reply:
Thanks a lot for the review. Sorry, I'm not sure if I understand correctly. 
Do you mean we can keep all the code in [TcgMorLockSmm.c], but only rename VariableHaveTcgProtocols() and make it return TRUE in [VariableStandaloneMm.c]?
What name do you suggest to rename VariableHaveTcgProtocols() to?

Reply:
> I have no objections to the PR as-is. However, given that the only purpose of `VariableHaveTcgProtocols()` appears to be this check, and the conditional update of the variable, we could also just return TRUE here, no? We might want to rename this function to make it more obvious what the narrow use-case is for which it exists.

Hi @ardbiesheuvel,
Thanks a lot for review.
Do you mean we can keep all the code in [TcgMorLockSmm.c], but only rename VariableHaveTcgProtocols() and make it return TRUE in [VariableStandaloneMm.c]?

Reply:
Based on the discussion in https://edk2.groups.io/g/devel/topic/106795434, I guess it's safe just to revert the original commit.